src/main/java/duke/command/HelpCommand.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.exception.*;
+import duke.task.*;
+import duke.ui.*;
+import java.util.List;
+
+public class HelpCommand extends Command<List<Task>> {
+    public HelpCommand() {
+        super("help", List.of());
+    }
+
+    @Override
+    public List<Task> execute(List<Task> tasks) {
+        System.out.printf("\nHere's all the things I can do for you! ~(^o.o^)\n");
```

Comment:
I like your introductory statements. It makes the user knows how to use your chat bot!